index.html

```diff
@@ -8,6 +8,83 @@
     <title>Code-Review</title>
 </head>
 <body>
-    
```

Comment:
Kindly Use main tag to improve the semantics of HTML Code. 👍🏻 
Happy Coding 😄 🌈